src/library_sdl.js

```diff
@@ -1052,10 +1052,8 @@ var LibrarySDL = {
 #if ASSERTIONS
       assert(tempCtx, 'TTF_Init must have been called');
 #endif
-      tempCtx.save();
       tempCtx.font = fontString;
       var ret = tempCtx.measureText(text).width | 0;
-      tempCtx.restore();
```

Comment:
why are these safe to remove? I don't remember exactly why they were needed, but we'd need to be sure that no other place could be influenced by that `.font` change?

Reply:
`SDL.ttfContext` is only used on this function, and it always set font, so I don't think any other code relies on the `font` value.